definitions/scenarios/upgrade_to_satellite_6_10.rb

```diff
@@ -54,7 +54,22 @@ def set_context_mapping
       context.map(:assumeyes, Procedures::Installer::Upgrade => :assumeyes)
     end
 
+    def check_var_lib_pulp
+      group_id = File.stat('/var/lib/pulp/').gid
+      if Etc.getgrgid(group_id).name != 'pulp'
+        raise "Please run 'foreman-maintain prep-6.10-upgrade' prior to upgrading."
+      end
+    end
+
     def compose
+      check_var_lib_pulp
+
+      add_step(Procedures::Service::Start)
+      add_step(Procedures::Service::Enable.
+          new(:only => Features::Pulpcore.pulpcore_migration_services))
+      add_step(Procedures::Service::Start.
+          new(:only => Features::Pulpcore.pulpcore_migration_services))
+      add_step(Procedures::Content::Switchover.new(:skip_deb => true))
```

Comment:
@upadhyeammit if this step fails, i need the entire operation to stop.  I don't think that happens today.  Is there an easy way to accomplish this?   Would moving this portion to the pre-migrations compose accomplish this?

Reply:
Can you try adding `run_strategy :fail_fast` for `Migrations` class in `upgrade_metadata`?

Reply:
Can you try adding `run_strategy :fail_fast` for `Migrations` class in `upgrade_metadata`?

Reply:
that worked, thank you!

Reply:
Instead of passing `:skip_deb` as `true` I think we better keep it true by default something like,
~~~
param :skip_deb, 'Do not run debian options in installer.', :flag => true, :default => true
~~~

Reply:
defining it true by default means that the normal switchover command needs to set it to false, which seems not ideal?   curious why it should be true by default?

Reply:
ah yes for upstream switchover we need to have the deb options and those are not required for this specific scenario! Agree to keep current implementation.